Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -138,23 +138,20 @@ GameListItem::GameListItem(const std::string& _rFileName)
 		m_Platform = DiscIO::IVolume::ELF_DOL;
 	}
 
-	if (!m_pImage.empty())
+	std::string path, name;
+	SplitPath(m_FileName, &path, &name, nullptr);
+	if (ReadPNGBanner(path + name + ".png"))
+	{ }
+	else if (ReadPNGBanner(path + "icon.png"))
+	{ }
```

Comment:
Yeah, that's what I meant. It can probably just be moved to its own function called IsPNGBannerPresent or something, and then just call that here to simplify things a little.